EngineFinder.cmake

```diff
@@ -1,12 +1,9 @@
-# {BEGIN_LICENSE}
```

Comment:
This line appears when I creating a new project. Is this correct?

Reply:
looks like is part of the template, however, I dont see a need to have that, if you create an issue in o3de I can remove it from the template